OIPA/iati/parser/validators.py

```diff
@@ -1530,6 +1538,22 @@ def activity_transaction(
                     apiField="transaction_type.code",
                     ))
 
+        if not disbursement_channel_code:
+            errors.append(
+                RequiredFieldError(
```

Comment:
Not sure how the new validator works so maybe I'm interpreting this wrong. @HatimKH disbursement_channel is optional for transactions, without looking at the structure it looks like this might make it mandatory?